src/components/Editor/index.js

```diff
@@ -525,10 +525,12 @@ class Editor extends PureComponent {
     const { sourceId } = selectedLocation;
 
     if (bp) {
+      // NOTE: it's possible the breakpoint has slid to a column
+      column = column || bp.location.column;
```

Comment:
Ah ok i see what is going on here. I think the comment should be more specific, maybe something like:

```
// falling back on the bp.location.column allows toggling the breakpoint a from the gutter, even if it is a column breakpoint
```